template_content/.gitignore

```diff
@@ -1,5 +1,9 @@
 # Git ignore - https://git-scm.com/docs/gitignore
 
+
+# Since this is a template, lets not commit the lock file
+poetry.lock
```

Comment:
this should go in the root .gitignore, otherwise users will be unable to commit the lock file in their generated template